diff --git a/browser/sql-parser.js b/browser/sql-parser.js
index a3cf81527a852e21f85506cc717407873f980cd9..ad47f928659176ffdcb11c97e30813470655da1d 100644
--- a/browser/sql-parser.js
+++ b/browser/sql-parser.js
@@ -1,3 +1,5 @@
+'use strict';
+
 (function(root) {
   var SQLParser = function() {
     function require(path){ return require[path]; }
@@ -583,15 +585,14 @@ parse: function parse(input) {
         vstack.length = vstack.length - n;
         lstack.length = lstack.length - n;
     }
-    _token_stack:
-        function lex() {
-            var token;
-            token = lexer.lex() || EOF;
-            if (typeof token !== 'number') {
-                token = self.symbols_[token] || token;
-            }
-            return token;
+    function lex() {
+        var token;
+        token = lexer.lex() || EOF;
+        if (typeof token !== 'number') {
+            token = self.symbols_[token] || token;
         }
+        return token;
+    }
     var symbol, preErrorSymbol, state, action, a, r, yyval = {}, p, len, newState, expected;
     while (true) {
         state = stack[stack.length - 1];
